documentation/sidebars.js

```diff
@@ -148,8 +148,48 @@ module.exports = {
         },
         {
           type: 'doc',
-          id: 'libraries/java/examples',
```

Comment:
You need to have an actual how to category, look at the rust version

Reply:
fixed